frontends/web/src/contexts/AppProvider.tsx

```diff
@@ -26,7 +26,8 @@ export const AppProvider = ({ children }: TProps) => {
   const [guideShown, setGuideShown] = useState(false);
   const [guideExists, setGuideExists] = useState(false);
   const [hideAmounts, setHideAmounts] = useState(false);
-
+  const [activeSidebar, setActiveSidebar] = useState(false);
+  const [sidebarStatus, setSidebarStatus] = useState('');
```

Comment:
lmk if we should define a type for `sidebarStatus` (to only allow '' and 'forceHidden').

Reply:
Yes please so I wont forget that we have forceHidden. I hope we can remove sidebarStatus at some point in the future.